samples/tools/webarena/README.md

```diff
@@ -0,0 +1,22 @@
+# WebArena Benchmark
+
+This directory is about the [WebArena](https://arxiv.org/pdf/2307.13854.pdf) benchmark in AutoGen.
+
+## Installation of WebArena
+
+WebArena can be installed by following the instructions from the [WebArena github](git@github.com:web-arena-x/webarena.git)
+
+If using WebArena with AutoGen there is a clash on the versions of OpenAI and some code changes are needed in WebArena to be compatible with AutoGen's OpenAI version.
```

Comment:
Can you explain more about the version issue? E.g., what is the mitigation that user would have to do, and what are the steps to get webarena installed properly.

Reply:
Olga, are you planning to release your scripts for installing WebArena, because their instructions are messy?

Reply:
@gagb I followed their instructions for installing webarena as a package, for the webpages @cheng-tan is the one to contact :)

Reply:
@ekzhu updated